src/components/OrgPicker/OrgPicker.tsx

```diff
@@ -125,7 +153,7 @@ export function OrgPicker({ single }: { single: Boolean }) {
         Current organization:
```

Comment:
As it happens I’m only in one organization, and weirdly I can still see the org picker!

<img width="503" alt="Screenshot 2024-09-26 at 2 02 11 PM" src="https://github.com/user-attachments/assets/e76a9462-4b5c-4b1a-9b21-660ac789255a">

Reply:
Whoa, great catch! This should be fixed now.